### ISSUE TITLE
Gemfile specified 2.4.0

### ISSUE THREAD

**Issue by trombonekenny** [mdwheele/classroom-docker#2]
`./classroom build` succeeds and I can go into `./classroom shell` but I get this error:

```
mussulma@endeavour:/usr/src/app$ ./script/setup 
==> Installing gem dependencies…
Your Ruby version is 2.4.2, but your Gemfile specified 2.4.0
```

**Comment by mdwheele** [comment]
Hmmmm, `cat Gemfile.lock` and tell me what you see for `RUBY VERSION`:

This is what I see.

```
RUBY VERSION
   ruby 2.4.2p198
```

**Comment by trombonekenny** [comment]
RUBY VERSION
   ruby 2.4.0p0

**Comment by trombonekenny** [comment]
It's from https://github.com/mdwheele/classroom/blob/7ada765524ba3f7cc1aea12227584fa10cd89953/Gemfile.lock#L503

**Comment by mdwheele** [comment]
Well that... is strange. Dumb question: could you verify you're on `feature/support-ghe`?

See: https://github.com/mdwheele/classroom/blob/feature/support-ghe/Gemfile.lock#L520

**Comment by mdwheele** [comment]
That said, I found the "bug" and have a workaround I need to send to that PR. 

The API endpoint is returning `204 No Content` on Enterprise and the correct `200 OK` with contents on GitHub.com (see https://developer.github.com/enterprise/2.11/v3/repos/collaborators/#response-1). 

I'm going to implement a workaround so we can get further and at the same time reach out to our representative with GitHub and see what the deal is. The feature is documented in the enterprise API documentation as "there")

**Comment by trombonekenny** [comment]
That was it. I had checked out a whole new copy for this `docker-compose` stuff and forgot to switch to the `feature/support-ghe` branch; I was still on `master`.

`script/setup` is working now (multiple tries for nokogiri)

**Comment by trombonekenny** [comment]
Good news on the bug hunt! Thanks for working at it!

**Comment by mdwheele** [comment]
No problem at all. I have a feeling the constant gap between GitHub and GitHub Enterprise regarding the API is going to become a bigger and bigger blocker. Hopefully, I can get a patch into Octokit for this, but if I were them, I don't think I would take something like this without a better "compat shim" for the library which will take more time.

**Comment by trombonekenny** [comment]
Any update on things? I'm getting ready to ping instructors doing a pilot with us, and I was hoping to give them a chance to try out Classroom via GHE. I guess we're waiting on upstream to look at the GHE API response problem?

**Comment by mdwheele** [comment]
Yeah, I'm waiting on GitHub to get back to me. I was able to reproduce the issue outside of Classroom or my PR. The only other thing I know to try is to attempt to detect the bugged output and, at that point, sift through all current invitations to find the `invitation.id` of the user being invited. That patch would have to be made in Octokit and then I would have to override Gemfile to pull from my fork rather than upstream. I think it's probably best to wait for a response from GitHub.
